content/en/tracing/guide/serverless_enable_aws_xray.md

```diff
@@ -1,174 +1,43 @@
-
-**Note:** The Datadog Lambda Layer and client libraries include the X-Ray SDK as a dependency, so you don't need to explicitly install it in your projects.
-
-Lastly, [install and import the X-Ray client library in your Lambda function](#installing-the-x-ray-client-libraries).
-
-#### Installing the X-Ray client libraries
-
-The X-Ray client library offers insights into your HTTP requests to APIs and into calls to DynamoDB, S3, MySQL and PostgreSQL (self-hosted, Amazon RDS, and Amazon Aurora), SQS, and SNS.
-
-Install the library, import it into your Lambda projects, and patch the services you wish to instrument.
-
-{{< programming-lang-wrapper langs="nodejs,python,go,ruby,java,.NET" >}}
-
-{{< programming-lang lang="nodejs" >}}
 
-Install the X-Ray tracing library:
+## Which one should you use?
 
-```bash
+### Use AWS X-Ray if:
 
-npm install aws-xray-sdk
+- You need a fast, lightweight way to get visibility into simple Serverless services.
+- You primarily use Lambda and API Gateway.
+- You're okay with AWS-managed sampling and limited trace retention.
+- You don't need access to payload data, custom tags, or trace/log correlation.
 
-# for Yarn users
-yarn add aws-xray-sdk
-```
+X-Ray is built into AWS and can be enabled with just a few clicks. It's well-suited for teams that need to debug basic issues or visualize service calls in low-complexity systems. However, you won't get function inputs and outputs, and the trace data is limited to what AWS provides. There's also no way to add custom metadata or correlate traces with logs. If you're debugging lightweight workflows and real-time gaps in trace visibility are acceptable, X-Ray is often "good enough."
 
-To instrument the AWS SDK:
+### Use Datadog APM if:
 
-```js
-var AWSXRay = require('aws-xray-sdk-core');
-var AWS = AWSXRay.captureAWS(require('aws-sdk'));
-```
+- You need full visibility into function inputs, outputs, and downstream calls.
+- Your system involves complex orchestration with services like Step Functions, SQS, DynamoDB, or third-party APIs.
+- You want to correlate traces with logs and metrics for faster debugging.
+- You need control over what data is retained and for how long, including full-fidelity traces.
 
-To instrument all downstream HTTP calls:
+Datadog APM goes far beyond what X-Ray offers. You can tag traces with business or customer context, trace asynchronous workflows end-to-end, and inspect payloads at each step. Sampling rules and retention policies are configurable, so you can reduce ingestion for high-throughput paths while preserving key traces for longer. This flexibility makes it ideal for growing systems and debugging hard-to-reproduce issues.
 
-```js
-var AWSXRay = require('aws-xray-sdk');
-AWSXRay.captureHTTPsGlobal(require('http'));
-var http = require('http');
-```
+## Feature comparison
 
-To instrument PostgreSQL queries:
+| Capability | AWS X-Ray | Datadog APM |
```

Comment:
Thoughts on this table? It does repeat the points from above but it can be easy to parse for readers